### ISSUE TITLE
Standalone Jackett doesn't start on Ubuntu 19.04

### ISSUE THREAD

**Issue by flightlevel** [Jackett/Jackett#5179]
Ubuntu 19.04 was recently released and with it dropped support for libssl1.0 which .NET Core relies on. Microsoft's github indicates that their next servicing release (usually monthly) will have a fix
[https://github.com/dotnet/core/issues/2526](https://tinyurl.com/y58upkvq)

**Comment by Forceflow** [comment]
So the solution is: wait for an update from MS? Fine by me.

**Comment by flightlevel** [comment]
Yep, wait for next .NET Core update, in the mean time can use docker or Mono version

**Comment by Mantene** [comment]
It looks like there is a fix for this - https://github.com/dotnet/core/issues/2526

**Comment by dagood** [comment]
.NET Core itself will find and use either 1.0 or 1.1, it's just the Debian package has an unnecessarily strict dependency. Installing from a tarball on 19.04 should work if you don't want to wait for a new Deb package.

**Comment by Forceflow** [comment]
> 
> 
> .NET Core itself will find and use either 1.0 or 1.1, it's just the Debian package has an unnecessarily strict dependency. Installing from a tarball on 19.04 should work if you don't want to wait for a new Deb package.

Are they actually planning on removing the restriction in the Deb package?

**Comment by dagood** [comment]
Yep, that change is in `master` for .NET Core 3.0 Preview 6 and onwards with https://github.com/dotnet/core-setup/pull/6070, and queued up for the next servicing releases of 2.1/2.2. It's changing to `|` dependencies that should work on all supported distros/releases.

**Comment by flightlevel** [comment]
Latest release supports Ubuntu 19

**Comment by Mantene** [comment]
Thanks! Works great!
